pkg/debug/checkups/server_data.go

```diff
@@ -0,0 +1,82 @@
+package checkups
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/kolide/launcher/pkg/agent/storage"
+	"github.com/kolide/launcher/pkg/agent/types"
+	"go.etcd.io/bbolt"
+)
+
+var serverProvidedDataKeys = []string{
+	"munemo",
+	"organization_id",
+	"device_id",
+	"remote_ip",
+	"tombstone_id",
+}
+
+type serverDataCheckup struct {
+	k       types.Knapsack
+	status  Status
+	summary string
+	data    map[string]any
+}
+
+func (sdc *serverDataCheckup) Data() map[string]any  { return sdc.data }
+func (sdc *serverDataCheckup) ExtraFileName() string { return "" }
+func (sdc *serverDataCheckup) Name() string          { return "Server Data" }
+func (sdc *serverDataCheckup) Status() Status        { return sdc.status }
+func (sdc *serverDataCheckup) Summary() string       { return sdc.summary }
+
+func (sdc *serverDataCheckup) Run(ctx context.Context, extraFH io.Writer) error {
+	db := sdc.k.BboltDB()
```

Comment:
This should not use BboltDB. (basically, almost nothing should)

You can get the KV store with `k.ServerProvidedDataStore()` and the `store.Get([]byte("device_id"))` or whatever

Reply:
We're trying to remove all bbolt from the codebase, and push it through kv interfaces